Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG PHP_VERSION
+ARG PHP_EXTENSIONS
+
+FROM php:${PHP_VERSION}-fpm
+
+WORKDIR /usr/local/etc/php/conf.d/
+COPY --link symfony.ini .
+WORKDIR /usr/local/etc/php/pool.d/
+COPY --link symfony.pool.conf .
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏You could use this instead IMO

```suggestion
WORKDIR /usr/local/etc/php
COPY --link symfony.ini ./conf.d/
COPY --link symfony.pool.conf ./pool.d/
```

Reply:
Done :+1: